docs/fencing/README.md

```diff
@@ -0,0 +1,41 @@
+- **Pacemaker** acts as the "brain," providing HA capabilities and managing resources through resource agents and recovery functions through fencing agents. Corosync provides the necessary networking and messaging components.
+  - **Etcd management by Pacemaker** is central to maintaining quorum in a two-node setup. Unlike normal OpenShift clusters where etcd runs as pods within the openshift-etcd namespace, in TNF, **Pacemaker runs etcd using Podman containers outside of the cluster**'s direct management.
+- **Corosync** is a core component of the RHEL-HA stack, working alongside Pacemaker. It is an open-source project that provides essential networking and messaging components for the cluster. 
+  - **Corosync is responsible for providing a consistent view of cluster membership, reliable ordered messaging, and flexible quorum capabilities**. In a two-node TNF cluster, Corosync **ensures that the nodes** can form a "C-quorum" (Corosync quorum) by establishing **membership with each other**
+---
+## Installation and Initialization 
+>Note: This process is handled via the provided [dev deployment guide](../../deploy/ipi-baremetalds-virt/README.md), but is described here for your information
+
+ The installation process starts with an initial boostrap node, which will then reboot into one of two final control plane nodes. A critical step involves the Cluster Etcd Operator (CEO), which, upon detecting the DualReplica control plane topology, triggers a TNF controller job. This controller runs pcs commands to initialize Pacemaker and configure the podman-etcd resource agent on both nodes. CEO then relinquishes control of etcd by setting specific flags, causing etcd containers to be removed from static pod configs and immediately recreated by Pacemaker using Podman. Fencing credentials, collected via the install-config (specifically RedFish details like address, username, and password), are made available to the nodes and used by Pacemaker to set up fencing. The Bare Metal Operator (BMO) is also adjusted to prevent power management of control-plane nodes in this topology to avoid conflicts with Pacemaker.
+
+## Disruption handling
+- **Recovery and State**: After a node reboots, it re-establishes membership with the etcd cluster, initially as a learner, and then becomes a fully joined member. The cluster ensures stability by maintaining a consistent cluster ID. While the cluster can operate in a degraded single-node state if one node fails, this is not recommended for long-term health, as it impacts operators and operations dependent on multiple nodes. See below for handling of graceful and ungraceful disruptions.
+---
+ - **Graceful Disruption Events**: Support as of version 4.19 of Openshift
```

Comment:
These bullet / headings did  not render the way you intended. You might need an extra space to force them to a new line. Alternatively, it may be best just to use headings to separate out the sections.